components/mpas-ocean/src/shared/mpas_ocn_tracer_ecosys.F

```diff
@@ -1894,7 +1897,10 @@ subroutine ocn_tracer_ecosys_surface_flux_compute(activeTracers, ecosysTracers,
       call marbl_instances(1)%surface_flux_compute()
 
       if (marbl_instances(1)%StatusLog%labort_marbl) &
-        call mpas_log_write("WARNING: problem in MARBL surface_flux_compute", MPAS_LOG_CRIT)
+! maltrud ignore abort warning
+!       call mpas_log_write("WARNING: problem in MARBL surface_flux_compute", MPAS_LOG_CRIT)
+! write(*,*)' IGNORING problem in MARBL surface_flux_compute'
+        marbl_instances(1)%StatusLog%labort_marbl = .false.
```

Comment:
Same here. We shouldn't add commented-out lines to the code. The reasoning should be explained, even if we decide to not call a critical error.

Reply:
Good point, @mark-petersen!  Unfortunately the current master doesn't run in BGC mode longer than a couple of years without these warnings flagging an abort.  So there is no baseline without them commented out.  If MARBL tracers go completely off the rails (i.e. NaNs), then the run still aborts as it should. The warnings need debugging, which is why I've left them commented and obvious.  Not sure when we'll get a chance though.  Is there a preferred comment I should add?

Reply:
Okay, will do.  Thanks